### PR TITLE
AR: clean-up code examples and add ERDs

### PR DESCRIPTION
**Pull request by nolds9**
@amaseda @adambray

### CODE REVIEW

File: 04-ruby-mvc-sinatra/active-record-intro/readme.md [hunk 1]
```diff
@@ -158,6 +163,9 @@ gem "pry"  # this gem allows access to REPL
 Then I'm going to run `$ bundle install` in the terminal.
 
 ### Setup Ruby - Tunr (You Do - 10 / 70)
+If you did not complete the initial SQL setup for Tunr above:
```
Comment (nolds9) [inline 04-ruby-mvc-sinatra/active-record-intro/readme.md]: should there be more steps?

File: 04-ruby-mvc-sinatra/active-record-intro/readme.md [hunk 2]
```diff
@@ -455,11 +462,13 @@ jesse.students.create(first_name: "baskin", last_name: "robbins", age: 34, job:
...
 Seeding a database is not all that different from the things we've been doing today. What's the purpose of seed data? **(ST-WG)**
 
-We want some sort of data in our database so that we can test our applications. Let's create a seed file in the terminal: `$ touch config/seeds.rb`
+We want some sort of data in our database so that we can test our applications. Let's create a seed file in the terminal: `$ touch db/seeds.rb`
 
-In our `config/seeds.rb` file let's put the following:
+In our `db/seeds.rb` file let's put the following:
 
 ```ruby
+require "bundler/setup" # require all the gems we'll be using for this app from the Gemfile. Obviates the need for `bundle exec`
```
Comment (nolds9) [inline 04-ruby-mvc-sinatra/active-record-intro/readme.md]: Will throw an error without

File: 04-ruby-mvc-sinatra/active-record-intro/readme.md [hunk 3]
```diff
@@ -314,7 +320,7 @@ george.destroy
 ## Associations
 
 ### Reframing (10 / 165)
-**Question:** We have a lot of choice when it comes to databases, why did we choose to use SQL?
+**Question:** We have a lot of choice when it comes to databases, are we using a SQL db in PostgreSQL?
```
Comment (nolds9) [inline 04-ruby-mvc-sinatra/active-record-intro/readme.md]: Thoughts on this question?

File: 04-ruby-mvc-sinatra/active-record-intro/readme.md [hunk 4]
```diff
@@ -314,7 +320,7 @@ george.destroy
...
```
Comment (amaseda) [inline 04-ruby-mvc-sinatra/active-record-intro/readme.md]: I think you meant to include "why" before "are we using..."

File: 04-ruby-mvc-sinatra/active-record-intro/readme.md [hunk 5]
```diff
@@ -113,7 +118,7 @@ CREATE TABLE students (
 now lets run our `wdi_schema.sql` file in the terminal:
 
 ```bash
-$ psql -d wdi_db < config/wdi_schema.sql
+$ psql -d wdi_db < db/wdi_schema.sql
```
Comment (adambray) [inline 04-ruby-mvc-sinatra/active-record-intro/readme.md]: nice switch to match other exercises!

File: 04-ruby-mvc-sinatra/active-record-intro/readme.md (hunk 4 above)
Comment (nolds9) [inline 04-ruby-mvc-sinatra/active-record-intro/readme.md]: True :+1:

File: 04-ruby-mvc-sinatra/active-record-intro/readme.md (hunk 3 above)
Comment (adambray) [inline 04-ruby-mvc-sinatra/active-record-intro/readme.md]: I'm not that into, in that I'm not exactly sure what it's asking (SQL vs NoSQL?) We talked a lot about that yesterday...

File: 04-ruby-mvc-sinatra/active-record-intro/readme.md (hunk 1 above)
Comment (adambray) [inline 04-ruby-mvc-sinatra/active-record-intro/readme.md]: I think this is correct, but I'm gonna ping @jshawl for clarification. For some reason I always have trouble in this scenario:

1. Students clone an exercise
2. I push a new branch after they've cloned
3. They need to checkout that branch.

File: 04-ruby-mvc-sinatra/active-record-intro/readme.md (hunk 3 above)
Comment (amaseda) [inline 04-ruby-mvc-sinatra/active-record-intro/readme.md]: Point here is to highlight that SQL is a **relational** database and use that as a segue into Active Record associations. Don't think the PostgreSQL distinction adds much here.

That being said, if this question doesn't sound like it's helpful @adambray, okay with axing it and just diving right into associations.

File: 04-ruby-mvc-sinatra/active-record-intro/readme.md (hunk 1 above)
Comment (jshawl) [inline 04-ruby-mvc-sinatra/active-record-intro/readme.md]: `git fetch https://github.com/ga-dc/tunr_sinatra` first to make sure all remote branches are fetched.

then `git checkout 2_active_record_starter`

**Pull request merged**